src/Ledger/Conway/Foreign/HSLedger/Core.agda

```diff
@@ -80,13 +80,8 @@ module Implementation where
   toData _ = 0
 
   ExUnits      = ℕ × ℕ
-  ExUnit-CommutativeMonoid = CommutativeMonoid 0ℓ 0ℓ ExUnits ∋ (Conversion.fromBundle record
-    { Carrier = ExUnits
-    ; _≈_ = _≈ᵖ_
-    ; _∙_ = _∙ᵖ_
-    ; ε = zero , zero
-    ; isCommutativeMonoid = pairOpRespectsComm +-0-isCommutativeMonoid
-    }) where open import Algebra.PairOp ℕ zero _≡_ _+_
+  ExUnit-CommutativeMonoid =
+    Conversion.fromBundle (commutativeMonoid +-0-commutativeMonoid +-0-commutativeMonoid)
```

Comment:
I'm not opposed to this change in principle, as I agree we should use stuff from stdlib or stdlib-classes when it simplifies... but I could understand what we had before at a glance, whereas using `Conversion.fromBundle` requires either familiarity with these conversions already or jumping to the `CommutativeModule.Core` module to see what's happening here... it's not terribly inconvenient, but does add an extra layer of abstraction.

Reply:
Ah, I see it's not set to auto-merge, so I'll approve it.